### ISSUE TITLE
No clear peak

### ISSUE THREAD

**Issue by HVHunt** [schatzlab/genomescope#96]
Good afternoon,

I have obtained this output from GenomeScope2.0:
http://qb.cshl.edu/genomescope/genomescope2.0/analysis.php?code=8R88KLKRwQtldaGTCBjx

This is for a diploid plant genome. Genome size is unknown. Input data was 150 bp Illumina reads (post-trimming) ran through Jellyfish with k=17 and -C flag. (I have also tried with higher k of 25, with similar results).

I thought the problem might be excessive coverage as on a previous similar post, but the above result is what I get even after downsampling to 5% of the original read number (which is only the R1 paired read set in any case). 

My questions are:
1) any advice on why no clear peak is seen?
2) here, it estimates genome size as c. 400 kbp. But when I ran the whole dataset (i.e. not downsampled) it was ~13 Mbp. And when I included the R2 reads it was ~26 Mbp. Can you explain why the estimated genome size seems to vary proportionally with the quantity of input data?

Many thanks, Harriet

**Comment by mschatz** [comment]
This is confusing and I wouldn’t trust any of the reported metrics. The
lack of a peak usually suggests that you have too low of coverage but you
seem to have pretty nice coverage and would mean really high Illumina
error. Can you try profiling without any trimming? Maybe something went
wrong there. And how large are the fastq files? For a plant I would expect
the genome size to be at least 100Mbp (e.g. Arabidopsis) and potentially
huge (many gigabases like wheat or pine). Do you have any relatives from
the same genus to give a clue?

Mike

**Comment by HVHunt** [comment]
Hi Mike,

Thanks for your reply. The genome size isn't known for any species in the genus, though there is a species in a sister genus, Ehertia, with a C-value estimate of 3.61. The Ehertia sp. is probably diploid; ploidy of my taxon is unknown, though could be hexaploid. 

I had 4 libraries PE-sequenced, each file is c. 13Gbytes. I have now worked out that low coverage (maybe because of large genome size?) was an issue, when I concatenated all trimmed read files (so ~102 Gbytes in total) I do get a peak:

http://qb.cshl.edu/genomescope/genomescope2.0/analysis.php?code=40jyGVRsHcQsShxck12C

Though I can only see one peak not two? The peak is still at quite low coverage compared to many of the examples on the Genomescope site... would this explain the apparent absence of additional peaks?

Also the error sequence curve seems to persist to higher coverage than usual - does this look like excessive Illumina error to you?

I also ran this dataset with the 6-ploid model:
http://qb.cshl.edu/genomescope/genomescope2.0/analysis.php?code=KGkkYbkDmTMNEb4ZRleX
This gives a better fit, I don't know if this supports the hypothesis of hexaploidy. 

I did also try your suggestion of running the raw reads, but that didn't help. Seems more likely that coverage was a problem, so the downscaling certainly wasn't helping.

**Comment by mschatz** [comment]
Yes, the core issue is the coverage is too low to get a good fit. By eye
the mode of the distribution is about 12x which is too low for a good
assembly or accurate inference, especially since we cant tell if that is
the homozygous peak or heterozygous peak, and we arent sure of the ploidy.
The hexaploid model looks overfit, and I cant get a good fit for diploid
even after some manual tweaks. With these data, my very crude estimate is
the genome size is at least a few gigabases but I cant be any more precise
than that. I know this is a big ask but Id strongly recommend going back to
do more sequencing.

Good luck
Mike
